cmd/cmd.go

```diff
@@ -117,26 +119,43 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if len(req.Files) > 0 {
-		fileMap := map[string]string{}
-		for f, digest := range req.Files {
+	var mu sync.Mutex
```

Comment:
Is this guarding the errgroup? I hope not. 

Be more clear about what it is guarding. Usually one defines directly below the mutex what it is guarding, visually grouped by blank lines. Grep stdlib for `sync.Mutex` for examples.

If g does not need mu held, then keep them separated, literally. If, in the rare case it is guarding something not declared where it, the mutex, is declared, then please document what it is guarding. For example:

```
var mu sync.Mutex // guards req.Files across goroutines below.
```

Reply:
The rules still holds, however, you don't need this lock. Read below.